client/src/main/java/com/alibaba/nacos/client/naming/net/HttpClient.java

```diff
@@ -16,6 +16,7 @@
 package com.alibaba.nacos.client.naming.net;
 
 import com.alibaba.nacos.api.common.Constants;
+import com.alibaba.nacos.client.config.common.HttpsManager;
```

Comment:
naming module shouldn't have a dependency of config module.

Reply:
naing and config package are both in the same moudule client

Reply:
> naming module shouldn't have a dependency of config module.

naing and config package are both in the same moudule client

Reply:
> > naming module shouldn't have a dependency of config module.
> 
> naing and config package are both in the same moudule client

I mean logically, not physically. Naming is a submodule for service discovery, while Config is another submodule for configuration management.

Reply:
> > > naming module shouldn't have a dependency of config module.
> > 
> > 
> > naing and config package are both in the same moudule client
> 
> I mean logically, not physically. Naming is a submodule for service discovery, while Config is another submodule for configuration management.

move them to cmmon pkg

Reply:
> > > > naming module shouldn't have a dependency of config module.
> > > 
> > > 
> > > naing and config package are both in the same moudule client
> > 
> > 
> > I mean logically, not physically. Naming is a submodule for service discovery, while Config is another submodule for configuration management.
> 
> move them to cmmon pkg

They should stay in client module, but in another package, may be: ``` com.alibaba.nacos.client.common```

Reply:
> > > > > naming module shouldn't have a dependency of config module.
> > > > 
> > > > 
> > > > naing and config package are both in the same moudule client
> > > 
> > > 
> > > I mean logically, not physically. Naming is a submodule for service discovery, while Config is another submodule for configuration management.
> > 
> > 
> > move them to cmmon pkg
> 
> They should stay in client module, and in another package, may be: ` com.alibaba.nacos.client.common`

okay